### ISSUE TITLE
Use Openpose with FLIR camera and Python API

### ISSUE THREAD

**Issue by ABesginow** [CMU-Perceptual-Computing-Lab/openpose#1279]
### Issue Summary

Use of flir cameras in combination with openpose python API

### Executed Command (if any)

`sudo python3 openpose_python.py `

I also tried the code with the following line:
`parser.add_argument("--flir_camera")`

### OpenPose Output (if any)

```
Starting OpenPose Python Wrapper...

Error:
No camera found.

Coming from:
- /home/guest/openpose/src/openpose/producer/producer.cpp:createProducer():460
- /home/guest/openpose/src/openpose/producer/producer.cpp:createProducer():471
- /home/guest/openpose/include/openpose/wrapper/wrapperAuxiliary.hpp:configureThreadManager():1188
- /home/guest/openpose/include/openpose/wrapper/wrapper.hpp:exec():424
- /home/guest/openpose/python/openpose/openpose_python.cpp:exec():223
Traceback (most recent call last):
  File "openpose_python.py", line 61, in <module>
    opWrapper.execute()
RuntimeError: 
Error:
No camera found.

Coming from:
- /home/guest/openpose/src/openpose/producer/producer.cpp:createProducer():460
- /home/guest/openpose/src/openpose/producer/producer.cpp:createProducer():471
- /home/guest/openpose/include/openpose/wrapper/wrapperAuxiliary.hpp:configureThreadManager():1188
- /home/guest/openpose/include/openpose/wrapper/wrapper.hpp:exec():424
- /home/guest/openpose/python/openpose/openpose_python.cpp:exec():223
```

### Type of Issue
You might select multiple topics, delete the rest:
- Help wanted
- Question

### Your System Configuration
1. **Whole console output** (if errors appeared), paste the error to [PasteBin](https://pastebin.com/) and then paste the link here: LINK

2. **OpenPose version**: Latest GitHub code? Or specific commit (e.g., d52878f)? Or specific version from `Release` section (e.g., 1.2.0)?
Latest Github version

3. **General configuration**:
    - **Installation mode**: Ubuntu via Cmake/Make
    - **Operating system** (`lsb_release -a` in Ubuntu):
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.6 LTS
Release:	16.04
Codename:	xenial
    - **Release or Debug mode**? (by default: release): Release
    - Compiler (`gcc --version` in Ubuntu or VS version in Windows): gcc (Ubuntu 5.4.0-6ubuntu1~16.04.11) 5.4.0 20160609

5. **3rd-party software**:
    - **Caffe version**: Default from OpenPose
    - **CMake version** (`cmake --version` in Ubuntu): 3.5.1
    - **OpenCV version**: 2.4.9.1 (latest I guess?)

8. If **Python** API:
    - **Python version**: 3.5
    - **Numpy version** (`python -c "import numpy; print numpy.version.version"` in Ubuntu): 1.16.4

I want to know how to use the FLIR cameras in combination with openpose, using the Python API provided.

The provided scripts only seem to run on images, can I just use "--flir_camera" as an argument? (Tried that, doesn't seem to work)
Also tried to use the FLIR Python API, the bad documentation results in the same issue.

In a nutshell: How do I get an image from the FLIR cameras in Python?

Thank you!

**Comment by ABesginow** [comment]
Found a way to get the FLIR camera image in Python.
Kudos to https://github.com/daan/calibrating-with-python-opencv
